CMSgov/mint-app
# [EASI-2899] Add AMS Model ID and other fields to General Characteristics

# EASI-2899

## Changes and Description

- Add fields to General Characteristics
  - AMS Model ID
  - Demo Code
  - Model Abbreviation
- Updated Store SQL to include new fields on General Characteristics
- There does not appear to be DB seeding for General Characteristics. That is outside the scope of the ticket so I have omitted it for now. This is a candidate for Tech Debt.



## How to test this change

- Run `scripts/dev up:backend && scripts/dev db:seed`
- Observe that the corresponding fields have been added to General Characteristics


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
